python/pypto/runtime/debug/__init__.py

```
@@ -0,0 +1,19 @@
+# -----------------------------------------------------------------------------------------------------------
+
+"""PyPTO runtime debug helpers.
+
+Not for production use. Modules under ``pypto.runtime.debug`` are intended
+for offline debugging of compiled artefacts (re-execution, tracing, etc.).
+"""
+
+from .replay import invalidate_binary_cache, replay
+from .run_script_writer import write_run_script
+
+__all__ = ["replay", "invalidate_binary_cache", "write_run_script"]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)